src/modules/0.0.23/aws_vpc/entity/route.ts

```diff
@@ -0,0 +1,74 @@
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+import { cloudId } from '../../../../services/cloud-id';
+import { RouteTable } from './route_table';
+
+@Entity()
+export class Route {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @ManyToOne(() => RouteTable, {
+    nullable: true, // shouldn't be, but orphanedRowAction doesn't work properly
+  })
+  @JoinColumn()
+  routeTable: RouteTable;
+
+  // destinations
+  @Column({ nullable: true })
+  @cloudId
+  DestinationCidrBlock?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  DestinationIpv6CidrBlock?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  DestinationPrefixListId?: string;
+
+  // through
+  @Column({ nullable: true })
+  @cloudId
+  EgressOnlyInternetGatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  GatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  InstanceId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  InstanceOwnerId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  NatGatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  TransitGatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  LocalGatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  CarrierGatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  NetworkInterfaceId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  VpcPeeringConnectionId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  CoreNetworkArn?: string;
+}
```

Comment:
It may make sense to drop `@cloudId` on this particular entity and define your own custom `entityId` and `generateId` methods. That way you can use the `routeTable.routeTableId` as part of the unique key.

Reply:
+1

Reply:
Actually if `Route` entity will be managed by `RouteTableMapper` you don't need the `@cloudId` concept because there will be no mapper to `cloud.read` a `Route`

Reply:
I suppose `generateId` should be used together with a mapper, right? But we don't have a mapper for this entity.